src/subprocess-posix.cc

```diff
@@ -16,6 +16,7 @@
 
 #include <sys/select.h>
 #include <assert.h>
+#include <atomic>
```

Comment:
nit: try separating C++ and C system headers into different blocks. I recommend using `git clang-format` do that automatically. Otherwise, push the `<atomic`>  include in a separate line below 22.